### ISSUE TITLE
Wrong project settings UI layout

### ISSUE THREAD

**Issue by vasily-kirichenko** [Microsoft/visualfsharp#3554]
![image](https://user-images.githubusercontent.com/873919/30063520-a59ea070-9257-11e7-9cfd-5cc302ef8861.png)

**Comment by KevinRansom** [comment]
oh ... high dpi bug,

Thanks

**Comment by vasily-kirichenko** [comment]
It's not on a high DPI display, it's on a ordinary FulHD one.

**Comment by KevinRansom** [comment]
Indeed and when I added it, it was on a hi-dpi screen.  I have no idea how to get the coordinates correct but I expect someone on the roslyn team has a clue.  Perhaps @brettfo might know.

**Comment by Pilchie** [comment]
@tannergooding made a bunch of changes here for C# and VB projects.

**Comment by tannergooding** [comment]
Fix should be here: https://github.com/Microsoft/visualfsharp/pull/3562, both of the labels being cut-off where not configured to auto-size.
